app/github_repository.rb

```diff
@@ -1,6 +1,6 @@
 require 'octokit'
 
-class GitHub
+class GitHubRepository
```

Comment:
I think this is "repository" as in "repository pattern"? Maybe this could be confused with a GitHub repo? Perhaps something like `GithubRepoFetcher` or something?